app/controllers/api/projects.php

```diff
@@ -138,6 +138,14 @@
 
         $databases = Config::getParam('pools-database', []);
 
+        if ($region !== 'default') {
```

Comment:
I think we need a more robust check here. The console sends the `region` attribute right? 

if the console stops sending the region or sends a different string, this check will break .

Reply:
We should test what happens  if region is empty and if its default